starknet_py/tests/e2e/contract_interaction/interaction_test.py

```diff
@@ -49,58 +49,34 @@ async def test_auto_fee_estimation(map_contract):
 
 
 @pytest.mark.asyncio
-async def test_throws_on_both_max_fee_and_auto_estimate(map_contract):
-    key = 2
-    value = 3
-
-    invocation = map_contract.functions["put"].prepare(key, value)
-    with pytest.raises(ValueError) as exinfo:
-        await invocation.invoke(max_fee=10, auto_estimate=True)
+async def test_throws(map_contract):
+    key, value = 2, 3
 
-    assert (
-        "Max_fee and auto_estimate are exclusive and cannot be provided at the same time."
-        in str(exinfo.value)
-    )
-
-
-@pytest.mark.asyncio
-async def test_throws_on_both_max_fee_in_prepare_and_auto_estimate(map_contract):
-    key = 2
-    value = 3
+    with pytest.raises(
+        ValueError,
+        match="Parameter max_fee must be specified when invoking a transaction.",
+    ):
+        await map_contract.functions["put"].invoke(key, value)
```

Comment:
This should still be their own separate tests. Combining them, you know nothing about what failed from just the test name.

Reply:
I mean, the error messages are really expressive already.
We could split it into 2 tests, one for each error message, as they are repeated here.
How does that sound?

Reply:
Two tests is better than one gigantic test, let's go for it.